### PR TITLE
Fully lint statsmodels/formula

### PR DESCRIPTION
**Pull request by jbrockmendel**
One by one...

### CODE REVIEW

File: statsmodels/formula/tests/test_formula.py [hunk 1]
```diff
@@ -74,25 +77,68 @@ def test_tests():
...
-    from pandas import read_table
-    dta = read_table(dta, sep=" ")
+    dta = StringIO('type" "income" "education" "prestige"\n'
```
Comment (bashtage) [inline statsmodels/formula/tests/test_formula.py]: You lost a quote here. SHould be '"type"

Comment (jbrockmendel) [reply statsmodels/formula/tests/test_formula.py]: Thanks.

**Comment by jbrockmendel** [comment]
rebased

**Comment by jbrockmendel** [comment]
@josef-pkt one push of the button away from never being bothered about this directory again.  (And notice I even rebased the way you prefer)

**Pull request closed without merging**